app/src/main/java/org/stepic/droid/notifications/StepikNotificationManagerImpl.kt

```diff
@@ -274,6 +274,7 @@ class StepikNotificationManagerImpl
             return
         }
         intent.action = AppConstants.OPEN_NOTIFICATION
+        intent.putExtra(AppConstants.KEY_NOTIFICATION_ID, id)
```

Comment:
At this class at least 5 times we set action for intent and putExtra with id. Let's extract it in the method.